### PR TITLE
Fix broken url in docs

### PR DESCRIPTION
**Pull request by saintmalik**
## Summary

Easy navigation and easy reading for others

### CODE REVIEW

File: content/github/site-policy/github-insights-and-data-protection-for-your-organization.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ Under various data protection regulations, such as the General Data Protection R
 
 - **Exclusion of processing**: Users may have the right to have their personal data excluded from being processed. Administrators have the ability to remove a contributor’s data from being processed in {% data variables.product.prodname_insights %}, and the resulting reports and metrics will exclude the contributor’s data accordingly. For more information, see "[Managing contributors and teams](/insights/installing-and-configuring-github-insights/managing-contributors-and-teams)."
 
-- **Access**: Users may have the right to demand to see what personal data is being processed. Each metric and report has a detailed description of what personal data is being processed. For more information, see "[Metrics available with {% data variables.product.prodname_insights %}](/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights)." Raw data is available through the {% data variables.product.prodname_enterprise %} API. Your organization is responsible for any decisions to process personal data and for fulfilling any such requests.
```
Comment (Jeff-GiJoe) [inline content/github/site-policy/github-insights-and-data-protection-for-your-organization.md:42]: best conservative cpa firms & emergency assistance for move to Oklahoma.

File: content/github/site-policy/github-insights-and-data-protection-for-your-organization.md [hunk 2]
```diff
@@ -39,8 +39,8 @@ Under various data protection regulations, such as the General Data Protection R
...
 
 - **Correction and deletion**: Users may have the right to rectify or delete their personal data. The data used in {% data variables.product.prodname_insights %} is derived from the existing data you add to or generate from your {% data variables.product.prodname_ghe_server %} installation. Correction and deletion should follow your organization's existing process to correct and delete data from {% data variables.product.prodname_ghe_server %}.
 
-- **Transparency regarding processing**:  Each metric and report has a detailed description of what personal data is being processed. For more information, see "[Metrics available with {% data variables.product.prodname_insights %}](/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights)."
+- **Transparency regarding processing**:  Each metric and report has a detailed description of what personal data is being processed. For more information, see "[Metrics available with {% data variables.product.prodname_insights %}](https://docs.github.com/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights)."
```
Comment (Jeff-GiJoe) [inline content/github/site-policy/github-insights-and-data-protection-for-your-organization.md:46]: Help

Comment (Jeff-GiJoe) [inline content/github/site-policy/github-insights-and-data-protection-for-your-organization.md:46]: Help

Comment (Jeff-GiJoe) [inline content/github/site-policy/github-insights-and-data-protection-for-your-organization.md:46]: Help
I don't understand the colors & meanings. Red, white,  black,  orange etc etc

**Review by Jeff-GiJoe — APPROVED** [review]
Thank you!!!!!

File: content/github/site-policy/github-insights-and-data-protection-for-your-organization.md (hunk 1 above)
Comment (saintmalik) [reply content/github/site-policy/github-insights-and-data-protection-for-your-organization.md:42]: I dont get you?

File: content/github/site-policy/github-insights-and-data-protection-for-your-organization.md (hunk 2 above)
Comment (saintmalik) [reply content/github/site-policy/github-insights-and-data-protection-for-your-organization.md:46]: Yeah, this /insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights leads to  https://docs.github.com/en/free-pro-team@latest/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights

but https://docs.github.com/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights leads to the actual page which is https://docs.github.com/en/enterprise-server@2.22/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights.

Comment (saintmalik) [reply content/github/site-policy/github-insights-and-data-protection-for-your-organization.md:46]: The green coloured text is the place that shows what changes was added, the red coloured text is the one that shows what was there before it was edited. the white lines show nothing was done there.

But wait are you new to this?

**Comment by saintmalik** [comment]
You are welcome

**Comment by heiskr** [comment]
Thanks for the pull request. 

- We don't support domains included in links because the site deploys a few different ways for different GitHub customers; that's the test failure on this branch. 
- Our translations we're investigating how to support pull requests to those, but right now they are one-way from a translation memory tool GitHub uses. So for the time being, we can't accept changes to the `translations/` folder in pull requests.

**Comment by saintmalik** [comment]
Okay,

But is there any way to get /insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights open up as https://docs.github.com/en/enterprise-server@2.22/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights.

instead of this https://docs.github.com/en/free-pro-team@latest/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights

**Comment by janiceilene** [comment]
👋 @saintmalik I'll talk with our site policy folks on exactly what fix to take with the broken link, but we're a bit swamped at the moment, so we're a little slow. Thanks for your patience 💖

**Comment by felicitymay** [comment]
Hi @saintmalik 👋🏻 - I'm not a member of site policy, but am a writer at GitHub so hope that I can help move these issues forwards. Thanks for raising this pull request. It highlights two different, interesting issues. Unfortunately, you happen to have picked two areas of the docs that have extra complexity: site policy (which has extra reviewing requirements) and translations (where we haven't got the process ready for external contributions yet).

With the broken links on "[GitHub Insights and data protection for your organization](https://docs.github.com/en/enterprise-server@2.22/github/site-policy/github-insights-and-data-protection-for-your-organization)" - it looks as if this site policy page is entirely about the GitHub Enterprise Server product "GitHub Insights." If you look at the Enterprise Server version of the page, the links work as expected.

I think that the underlying bug here is probably that this page shouldn't be displayed if you're looking at the docs for GitHub.com (Free, Pro, Team). I'll double-check with the site policy team and, if they agree, then we can update the front matter for the topic to fix this.

Thank you so much for pointing out the broken link in the Chinese translation too. I'll need to check with our localization team whether this is okay to fix here (since it's not translated material), or whether this needs to be feed back to the translations team so that your fix doesn't get overwritten by their next updates.

**Comment by felicitymay** [comment]
I've assigned myself to your pull request so that other people working on our reviewing board can see that this is in progress. I'll report back as soon as I hear from the site policy and localization teams.

**Comment by felicitymay** [comment]
> I'll need to check with our localization team whether this is okay to fix here (since it's not translated material), or whether this needs to be feed back to the translations team so that your fix doesn't get overwritten by their next updates.

The localization team has confirmed that this will need to be fixed in by the translation team in Crowdin, otherwise your fix is likely to be lost. I've opened an internal issue to tell them about the bad link and the fix that you've identified for it.

**Comment by saintmalik** [comment]
> I've assigned myself to your pull request so that other people working on our reviewing board can see that this is in progress. I'll report back as soon as I hear from the site policy and localization teams.

Thanks

**Comment by saintmalik** [comment]
> I've assigned myself to your pull request so that other people working on our reviewing board can see that this is in progress. I'll report back as soon as I hear from the site policy and localization teams.

Thanks

**Comment by saintmalik** [comment]
> > I'll need to check with our localization team whether this is okay to fix here (since it's not translated material), or whether this needs to be feed back to the translations team so that your fix doesn't get overwritten by their next updates.
> 
> The localization team has confirmed that this will need to be fixed in by the translation team in Crowdin, otherwise your fix is likely to be lost. I've opened an internal issue to tell them about the bad link and the fix that you've identified for it.

Glad to hear that

**Comment by felicitymay** [comment]
Hi @saintmalik - I'm talking to the site policy folks now about the best solution for the second broken link you reported. Hopefully we'll have a fix soon, but this will need to be fixed by an internal PR.

I suggest that we close this PR because, with our current set up, these problems can't be fixed in this repository. Even though I'm closing this PR, we're still very grateful that you highlighted these two broken links and gave us the opportunity to fix them (hence the `hacktoberfest-accepted` label). ✨ 🙇🏻‍♀️

**Comment by saintmalik** [comment]
> Hi @saintmalik - I'm talking to the site policy folks now about the best solution for the second broken link you reported. Hopefully we'll have a fix soon, but this will need to be fixed by an internal PR.
> 
> I suggest that we close this PR because, with our current set up, these problems can't be fixed in this repository. Even though I'm closing this PR, we're still very grateful that you highlighted these two broken links and gave us the opportunity to fix them (hence the `hacktoberfest-accepted` label). ✨ 🙇🏻‍♀️

Thanks 😊

**Comment by felicitymay** [comment]
It turns out that I was wrong and that the policy change can be made in a public repository, just not this one. (I'm still relatively new to GitHub and haven't needed to do anything much with policies before.)

I've opened a PR with what I suspect is the best fix (because it ensures that any new links added to the topic should work) here: https://github.com/github/site-policy/pull/325.

**Comment by felicitymay** [comment]
🎉 Good news. 

Your fix for the broken link in the Chinese version of [Caching your GitHub credentials in Git](https://docs.github.com/en/free-pro-team@latest/github/using-git/caching-your-github-credentials-in-git) has made it into the translation system. It may take up to a week for the fix to be visible on the docs site, but is definitely on its way.

The fix to show the [GitHub Insights and data protection for your organization](https://docs.github.com/en/enterprise-server@2.22/github/site-policy/github-insights-and-data-protection-for-your-organization) article only for people browsing the docs for GitHub Enterprise Server, which fixes the broken links you flagged, was just merged and is now live.

**Pull request closed without merging**